common/source/docs/common-multiple-rx.rst

```diff
@@ -0,0 +1,31 @@
+.. _common-multiple-rx:
+
+================================
+Multiple Radio Control Receivers
+================================
+
+It is possible to use multiple radio control receivers in ArduPilot for redundancy or hand-off between multiple transmitters.
+
+Configuration
+=============
+
+The additional receivers can be attached to any Serial Port's RX input as long as they use a serial protocol.
+These are: SBus, DSM, DSM2, DSM-X, IBus, SUMD, and SRXL.
+
+For whichever Serial Port the receiver is attached, the ``SERIALx_PROTOCOL`` should be set to 23. The baud rate is ignored and auto-detected, as well as the type of the receiver.
+
+Failsafe and Fall Over
```

Comment:
is it "Fall Over" or "Failover" I wonder..?

Reply:
will change to "changeover"...thinks its more correct...